README.md

```diff
@@ -126,28 +126,28 @@ Tagging a Release
 2. Create a new branch for the minor version (if not present):
 
 ```
-$ git checkout -b releases/1.9.x
+$ git checkout -b releases/2.1.x
 ```
 
 3. Update the hardcoded version tag in ``Client.php``:
 
 ```php
 class Raven_Client
```

Comment:
This class does not exists anymore in `2.x` and should be updated to reflect PSR-4 changes